networklibrary/build.gradle

```diff
@@ -35,5 +35,11 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
-    implementation 'com.github.Cogno-IDU:smslibrary:v-2.0'
+    implementation 'com.github.Cogno-IDU:smslibrary:v2.2'
```

Comment:
We should change the naming convention to numbers only so that android studio warns you when there's a new version. Not related to this PR, just a side note I found out.